app/view/css/ckeditor-contents.css

```diff
@@ -61,12 +61,47 @@ ol,ul,dl
 	padding: 0 40px;
 }
 
+p, li
+{
+	font-size: 12px;
```

Comment:
Should be `14px`. ;-)

("inspected" it in the before situation. It was 14px, and 12 would make it too small)